src/Shared/Server/infra/log.go

```diff
@@ -20,5 +20,13 @@ func (handler LogHandle) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 		log.Fatalln("http server: log handle:", err)
 	}
 
-	handler.logEnable <- (string(body) == "enable")
+	go func() {

```

Comment:
This is more related to the logger, but I'll put it here. The way the logger is setup right now, multiple logging sessions can be started at the same. Either we add ids to "logging sessions" so that multiple logging sessions can be coordinated at the same time or we set it up so that logging is shared between users. I'd go for the first options.

Reply:
currently we can only start a global logging session, multiple sessions can't be running at the same time because we only instantiate one logger and we use only one channel to toggle it. All routines receive the same channel and any one connected to the backend can start/stop the logger.

FW stated that no more than one person will be logging at the same time, so I wouldn't change it until they request it, specially if we want to deliver a new MVP soon (we need at least this week to test everything and put it all together).